packages/vuetify/src/composables/position.ts

```diff
@@ -0,0 +1,52 @@
+// Utilities
+import type { PropType } from 'vue'
+import { computed } from 'vue'
+import { convertToUnit } from '@/util/helpers'
+import propsFactory from '@/util/propsFactory'
+
+// Types
+export interface PositionProps {
+  bottom?: boolean | string
+  left?: boolean | string
+  position?: 'static' | 'relative' | 'fixed' | 'absolute' | 'sticky'
+  right?: boolean | string
+  top?: boolean | string
+}
+
+const positionValues = ['static', 'relative', 'fixed', 'absolute', 'sticky'] as const
```

Comment:
This could be used as the basis for the `position` property in PositionProps as well